### PR TITLE
Use vagga on github CI

### PR DESCRIPTION
**Pull request by anti-social**
Tested cargo and vagga caches inside my fork: https://github.com/anti-social/vagga/runs/4776366943

We could add `vagga _version_hash rust-musl` to the caches keys. What do you think?

### CODE REVIEW

File: vagga.yaml [hunk 1]
```diff
@@ -70,13 +70,13 @@ containers:
...
     - !BuildDeps [wget]
-    - !Install [make, curl, zip, file, git, parallel]
+    - !Install [make, curl, zip, file, git, ca-certificates, parallel]
     - !Sh |
         set -ex
         cd /tmp
-        git clone git://github.com/bats-core/bats-core
-        git clone git://github.com/bats-core/bats-support.git /bats/bats-support
-        git clone git://github.com/bats-core/bats-assert.git /bats/bats-assert
+        git clone --depth=1 https://github.com/bats-core/bats-core.git
```
Comment (tailhook) [inline vagga.yaml:77]: The `.git` used to be optional, isn't it?

**Review by tailhook — APPROVED** [review]
> We could add `vagga _version_hash rust-musl` to the caches keys. What do you think?

Yes, sounds good. This will make cache smaller (i.e. only the latest image version will be in the cache, not all versions that happened to be there until some inadvertent cache clean).

**Comment by anti-social** [comment]
Also I think about making a separate cache for project's vagga commands. So when we'll update rust version test cache won't be ivalidated.

**Comment by tailhook** [comment]
> Also I think about making a separate cache for project's vagga commands. So when we'll update rust version, tests cache won't be ivalidated.

Well, updating rust isn't frequent operation. So if it's not single-line fix or similar, I wouldn't bother.

**Comment by anti-social** [comment]
Often see the following error on the github CI: 
```
E: Failed to fetch https://ubuntu.zerogw.com/pool/main/v/vagga/vagga_0.8.1-83-gd1f1466-1_amd64.deb  Could not handshake: The TLS connection was non-properly terminated. [IP: 148.251.14.186 443]
```

What could the reason for this?

**Comment by anti-social** [comment]
@tailhook Could you review and merge if everything is ok

**Review by tailhook — APPROVED** [review]

**Comment by tailhook** [comment]
Sure. Merged!

**Pull request merged**